src/vivarium_gates_nutrition_optimization/data/extra_gbd.py

```diff
@@ -22,11 +20,10 @@ def load_lbwsg_exposure(location: str):
         location_id=location_id,
         sex_id=gbd_constants.SEX.MALE + gbd_constants.SEX.FEMALE,
         age_group_id=164,  # Birth prevalence
-        gbd_round_id=gbd_constants.ROUND_IDS.GBD_2019,
-        decomp_step=gbd_constants.DECOMP_STEP.STEP_4,
+        release_id = gbd_constants.RELEASE_IDS.GBD_2021,
     )
     # This data set is big, so let's reduce it by a factor of ~40
-    data = data[data["year_id"] == 2019].drop(columns="year_id")
+    data = data[data["year_id"] == 2021].drop(columns="year_id")
```

Comment:
Instead of this step we could just add a year_id line to the get_draws call? Then we wouldn't have to wait for all of the years to load?

Reply:
Using the vi_utils.get_draws function is essentially the same as calling get_draws from db_queries.get_draws, so here we can just specify the year_id.